### PR TITLE
vs-code: link from nav and key docs

### PR DESCRIPTION
**Pull request by jorgeorpinel**
- [x] External sidebar entries both at the top level (under Changelog) and under /doc/install
- [x] Link from all docs sections as appropriate
- [x] Other small changes related to ☝🏼 & needed redirects

### CODE REVIEW

File: content/docs/sidebar.json [hunk 1]
```diff
@@ -522,6 +522,11 @@
     "url": "https://github.com/iterative/dvc/releases",
     "type": "external"
   },
+  {
+    "label": "VS Code Extension",
```
[the remark this answers is not in the document]
Comment (jorgeorpinel) [reply content/docs/sidebar.json:534]: Thanks Julie. @shcheklein I don't think we can use their icon here at all actually.

image

> https://code.visualstudio.com/brand

[the remark this answers is not in the document]
Comment (shcheklein) [reply content/docs/sidebar.json:534]: I think it's fine in this case.

**Comment by jorgeorpinel** [comment]
I'm just not sure whether to link from the DVCLive section and if so from where exactly. Ideas @daavoo ? Thanks

File: content/docs/sidebar.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (jorgeorpinel) [reply content/docs/sidebar.json:534]: I don't think so. They only allow to associate/link the icon with VS Code itself. We would be using it to identify our own VS Code extension and link to the marketplace, not to the VS Code home page.

image

**Pull request merged**